app/services/participants/change_schedule/validate_and_change.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Participants
+  module ChangeSchedule
+    module ValidateAndChange
```

Comment:
Can we call it `ValidateAndChangeSchedule`? So that we don't get confused with other `ValidateAndChange` modules?

Reply:
Sure. I'll also rename the other to ValidateAndChangeState